### PR TITLE
Update Image and website url

### PR DESCRIPTION
**Pull request by yiranwu0**
## Why are these changes needed?

1. Update url of  cq-math and stateflow notebook, so that images can be shown in github repo.
  2. Change all `https://github.com/microsoft` to `https://github.com/autogen-ai`

## Checks

- [ ] I've included any doc changes needed for https://autogen-ai.github.io/autogen/. See https://autogen-ai.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by marklysze — APPROVED** [review]
Tested a few pages and looks good, thanks @yiranwu0

File: notebook/agentchat_RetrieveChat.ipynb [hunk 1]
```diff
@@ -287,7 +287,7 @@
       "\n",
       "#### Model List\n",
       "\n",
-      "- `lgbm_spark`: The class for fine-tuning Spark version LightGBM models, using [SynapseML](https://microsoft.github.io/SynapseML/docs/features/lightgbm/about/) API.\n",
+      "- `lgbm_spark`: The class for fine-tuning Spark version LightGBM models, using [SynapseML](https://autogen-ai.github.io/SynapseML/docs/features/lightgbm/about/) API.\n",
```
Comment (sonichi) [inline notebook/agentchat_RetrieveChat.ipynb]: remove this change, and a few similar ones below.

File: notebook/agentchat_RetrieveChat_pgvector.ipynb [hunk 2]
```diff
@@ -339,7 +339,7 @@
       "\n",
       "#### Model List\n",
       "\n",
-      "- `lgbm_spark`: The class for fine-tuning Spark version LightGBM models, using [SynapseML](https://microsoft.github.io/SynapseML/docs/features/lightgbm/about/) API.\n",
+      "- `lgbm_spark`: The class for fine-tuning Spark version LightGBM models, using [SynapseML](https://autogen-ai.github.io/SynapseML/docs/features/lightgbm/about/) API.\n",
```
Comment (sonichi) [inline notebook/agentchat_RetrieveChat_pgvector.ipynb]: same

File: notebook/agentchat_RetrieveChat_qdrant.ipynb [hunk 3]
```diff
@@ -276,13 +276,13 @@
       "    <br>\n",
       "</p>\n",
       "\n",
-      ":fire: Heads-up: We have migrated [AutoGen](https://microsoft.github.io/autogen/) into a dedicated [github repository](https://github.com/microsoft/autogen). Alongside this move, we have also launched a dedicated [Discord](https://discord.gg/pAbnFJrkgZ) server and a [website](https://microsoft.github.io/autogen/) for comprehensive documentation.\n",
+      ":fire: Heads-up: We have migrated [AutoGen](https://autogen-ai.github.io/autogen/) into a dedicated [github repository](https://github.com/microsoft/autogen). Alongside this move, we have also launched a dedicated [Discord](https://discord.gg/pAbnFJrkgZ) server and a [website](https://autogen-ai.github.io/autogen/) for comprehensive documentation.\n",
```
Comment (sonichi) [inline notebook/agentchat_RetrieveChat_qdrant.ipynb]:
```suggestion
      ":fire: Heads-up: We have migrated [AutoGen](https://autogen-ai.github.io/autogen/) into a dedicated [github repository](https://github.com/autogen-ai/autogen). Alongside this move, we have also launched a dedicated [Discord](https://discord.gg/pAbnFJrkgZ) server and a [website](https://autogen-ai.github.io/autogen/) for comprehensive documentation.\n",
```

File: notebook/agentchat_RetrieveChat_qdrant.ipynb [hunk 4]
```diff
@@ -296,7 +296,7 @@
...
       "- It supports fast and economical automatic tuning (e.g., inference hyperparameters for foundation models, configurations in MLOps/LMOps workflows, pipelines, mathematical/statistical models, algorithms, computing experiments, software configurations), capable of handling large search space with heterogeneous evaluation cost and complex constraints/guidance/early stopping.\n",
       "\n",
-      "FLAML is powered by a series of [research studies](https://microsoft.github.io/FLAML/docs/Research/) from Microsoft Research and collaborators such as Penn State University, Stevens Institute of Technology, University of Washington, and University of Waterloo.\n",
+      "FLAML is powered by a series of [research studies](https://autogen-ai.github.io/FLAML/docs/Research/) from Microsoft Research and collaborators such as Penn State University, Stevens Institute of Technology, University of Washington, and University of Waterloo.\n",
```
Comment (sonichi) [inline notebook/agentchat_RetrieveChat_qdrant.ipynb]: For FLAML, this link is invalid and it's better to keep the original link for now. There are similar cases below.

**Review by sonichi — CHANGES_REQUESTED** [review]
Could you please resolve the conflicts?

**Comment by yiranwu0** [comment]
@Hk669 Can you take a look at this PR? I saw you made the same changes in PR #18 and already merged it

**Comment by Hk669** [comment]
> @Hk669 Can you take a look at this PR? I saw you made the same changes in PR #18 and already merged it

looks good to me. please merge it. thanks @yiranwu0

**Review by Hk669 — APPROVED** [review]

**Review by sonichi — APPROVED** [review]

**Pull request merged**